test/migrations/20211019192121_create_cdn_configuration_katello_test.rb

```diff
@@ -1,38 +0,0 @@
-require 'katello_test_helper'
```

Comment:
Can you help me understand why this test is deleted?

Reply:
It sporadically caused errors like
```
[](https://ci.theforeman.org/blue/organizations/jenkins/katello-pr-test/detail/katello-pr-test/10204/pipeline/#step-132-log-1081)[](https://ci.theforeman.org/blue/organizations/jenkins/katello-pr-test/detail/katello-pr-test/10204/pipeline/#step-132-log-1082)[](https://ci.theforeman.org/blue/organizations/jenkins/katello-pr-test/detail/katello-pr-test/10204/pipeline/#step-132-log-1083)[](https://ci.theforeman.org/blue/organizations/jenkins/katello-pr-test/detail/katello-pr-test/10204/pipeline/#step-132-log-1084)[](https://ci.theforeman.org/blue/organizations/jenkins/katello-pr-test/detail/katello-pr-test/10204/pipeline/#step-132-log-1085)[](https://ci.theforeman.org/blue/organizations/jenkins/katello-pr-test/detail/katello-pr-test/10204/pipeline/#step-132-log-1086)[2022-02-17T20:03:36.851Z] Actions::Katello::Repository::UpdateRedhatRepository#test_0001_plans:

[2022-02-17T20:03:36.851Z] RuntimeError: Wrapped undumpable exception for: ActiveRecord::PreparedStatementCacheExpired: ERROR:  cached plan must not change result type

[2022-02-17T20:03:36.851Z] 

[2022-02-17T20:03:36.851Z]     /home/jenkins/workspace/katello-pr-test/app/models/katello/product.rb:109:in `cdn_configuration'

[2022-02-17T20:03:36.851Z]     /home/jenkins/workspace/katello-pr-test/app/models/katello/glue/pulp/repos.rb:127:in `repo_url'

[2022-02-17T20:03:36.851Z]     /home/jenkins/workspace/katello-pr-test/test/actions/katello/repository/update_redhat_repository_test.rb:19:in `block (2 levels) in <module:Actions>'

[2022-02-17T20:03:36.851Z]
```

Reply:
On several runs with the migration test I have seen that jenkins 100% succeeded only once. If run individually the test runs ok, but it seems to leave tests that run after in a questionable state.